buildlib/azure-pipelines-pr.yml

```
@@ -300,12 +301,12 @@ stages:
               exit 0;
             displayName: Test ucx_info
           - bash: |
-              set +eE
-              set -x
-              export LD_LIBRARY_PATH=$(System.DefaultWorkingDirectory)/ucx_bin_$(Build.BuildId)/install/lib:$LD_LIBRARY_PATH
-              $(System.DefaultWorkingDirectory)/ucx_bin_$(Build.BuildId)/install/bin/ucx_perftest -t tag_bw &
+              set +eE -x
```

why ignore errors here?